spec/factories/receptacles.rb

```diff
@@ -2,5 +2,6 @@
 
 FactoryBot.define do
   factory :receptacle do
+    labware { nil }
```

Comment:
I suspect this wasn't required before because its normally referenced the other way round where an asset as a receptacle?

Reply:
I don't think this was "necessary" exactly as it works without it, but I felt that it gave a hint to users of the factory that you can assign a labware if you want to.  Which is what I wanted to do.